moto/dynamodb/parsing/tokens.py

```diff
@@ -17,6 +17,7 @@ class Token:
     CLOSE_ROUND_BRACKET = ")"
     COMMA = ","
     SPACE = " "
+    NEW_LINE = "\n"
     DOT = "."
```

Comment:
Unrelated with the issue, but I noticed `EQUAL_SIGN` on line [15](https://github.com/getmoto/moto/pull/7128/files#diff-6960f4c8844f5f277906ab6c10516d64801261a18046cd4a873255d2d25b8ecdR15) and [29](https://github.com/getmoto/moto/pull/7128/files#diff-6960f4c8844f5f277906ab6c10516d64801261a18046cd4a873255d2d25b8ecdR29) can be deleted since they aren't used anywhere throughout the moto repository (it's a duplicate of `SPACE`).

Reply:
Good catch @abbasyadollahi - I've removed the field. Thanks for the review!